smoke/features/data/shareconfigmap.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
```
$ shellcheck shareconfigmap.sh 

In shareconfigmap.sh line 17:
create_shared_configmap ${1}
                        ^--^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
create_shared_configmap "${1}"

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```